.github/workflows/doc-publish.yml

```diff
@@ -121,6 +120,10 @@ jobs:
       - name: Extract live doc
         run: unzip livedoc.zip -d doc && unzip doc/livedoc.zip -d doc/doc
 
+      - name: Get REF_NAME
+        run: |
+          echo "REF_NAME=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+
```

Comment:
but didn't we conditionalize on refs/tags?

Reply:
Right. Done. Could I do this without making another step "Get tag"?